### PR TITLE
Added bluetooth demo with CC2650 SensorTag device

### LINKED ISSUE #20: Add KubeEdge demo with bluetooth protocol
Demonstrate the bluetooth protocol support of kubeEdge

### PR DESCRIPTION
**Pull request by sujithsimon22**
What type of PR is this?
/kind enhancement

What this PR does / why we need it
This PR adds a demo to demonstrate the ability of kubeEdge to support bluetooth protocol

Which issue(s) this PR fixes
Fixes #20

### CODE REVIEW

**Comment by sujithsimon22** [comment]
cc @rohitsardesai83 @sids-b @m1093782566

File: bluetooth-CC2650-demo/README.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
+# Bluetooth With KubeEdge Using CC2650
+
+
+Users can make use of KubeEdge platform to connect and control their bluetooth devices, provided, the user is aware of the of the data sheet information for their device.
+Kubernetes Custom Resource Definition (CRD) and KubeEdge bluetooth mapper is being used to support this feature, using which users can control their device from the cloud. Texas Instruments CC2650 SensorTag device is being shown here as an example.
```
Comment (rohitsardesai83) [inline bluetooth-CC2650-demo/README.md]: It's good if we can give a link to the CC2650 device product page or datasheet for users to check out.

File: bluetooth-CC2650-demo/README.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
+
+## Description
+
+KubeEdge support for bluetooth protocol has been demonstrated here by making use of Texas Instruments CC2650 SensorTag device.
+This section contains instructions on how to make use of bluetooth mapper of kubeedge to control CC2650 SensorTag device.
```
Comment (rohitsardesai83) [inline bluetooth-CC2650-demo/README.md]: kubeedge -> KubeEdge

File: bluetooth-CC2650-demo/README.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+  ```
+  
+  The bluetooth mapper has the following major components :-
+   - Action Manager
+   - Scheduler
+   - Watcher
+   - Controller
+   - Data Converter
+  
+  Please click [Bluetooth_Mapper](https://github.com/kubeedge/kubeedge/blob/master/device/bluetooth_mapper/README.md) to understand the features and  the usage of bluetooth mapper.
```
Comment (rohitsardesai83) [inline bluetooth-CC2650-demo/README.md]: More details on bluetooth mapper can be found [here](https://github.com/kubeedge/kubeedge/blob/master/device/bluetooth_mapper/README.md)

File: bluetooth-CC2650-demo/README.md [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+                   cd $GOPATH/src/github.com/kubeedge/examples/bluetooth-CC2650-demo/sample-crds
+                   kubectl apply -f CC2650-device-model.yaml
+                   kubectl apply -f CC2650-device-instance.yaml
+
+                   Note: You can change the CRDs to match your requirement
+        ```
+ 
+4. Please ensure that bluetooth service of your device is ON
+
+5. Set 'bluetooth=true' label for the node (This label is a prerequisite for the scheduler to schedule bluetooth_mapper pod on the node)
```
Comment (rohitsardesai83) [inline bluetooth-CC2650-demo/README.md]: can we add more details , like hardware / software stack supported node can be only used to deploy etc ?

**Comment by rohitsardesai83** [comment]
@sujithsimon22 , please fix minor review comments.

File: bluetooth-CC2650-demo/README.md (hunk 2 above)
Comment (sujithsimon22) [reply bluetooth-CC2650-demo/README.md]: Done, thanks

File: bluetooth-CC2650-demo/README.md (hunk 3 above)
Comment (sujithsimon22) [reply bluetooth-CC2650-demo/README.md]: Done, thanks

File: bluetooth-CC2650-demo/README.md (hunk 1 above)
Comment (sujithsimon22) [reply bluetooth-CC2650-demo/README.md]: Done, thanks

File: bluetooth-CC2650-demo/README.md (hunk 4 above)
Comment (sujithsimon22) [reply bluetooth-CC2650-demo/README.md]: The required hardware and software stack required has been mentioned in the Prerequisites section. Is that sufficient or should it be repeated here again?

Comment (rohitsardesai83) [reply bluetooth-CC2650-demo/README.md]: This label is a prerequisite for the scheduler to schedule bluetooth_mapper pod on the node  -->

This label is a prerequisite for the scheduler to schedule bluetooth_mapper pod on the node  [  which meets the hardware / software prerequisites ]

Comment (sujithsimon22) [reply bluetooth-CC2650-demo/README.md]: Done, thanks

**Pull request merged**